lib/Doctrine/ORM/PersistentCollection.php

```diff
@@ -121,9 +107,10 @@
      */
     public function __construct(EntityManagerInterface $em, $class, $coll)
     {
-        $this->coll      = $coll;
-        $this->em        = $em;
-        $this->typeClass = $class;
+        $this->collection  = $coll;
+        $this->em          = $em;
+        $this->typeClass   = $class;
+        $this->initialized = true;
```

Comment:
@Ocramius keeping same code as it was before. Look at deleted line 106 at same file. =)